lib/types/tests.ts

```diff
@@ -0,0 +1,164 @@
+import {
```

Comment:
These tests are not real tests. It's just pseudocode for typechecking.

Reply:
no problem.